### DISCUSSION TITLE
Libbi?

### DISCUSSION THREAD

**Discussion by JeremyRoebuck85** [CJNE/ha-myenergi#428] [Ideas]
Is anyone able to do the necessary to add the Myenergi Libbi battery? I'd really like to have SOC (state of charge) on my dashboard. Way beyond my own capabilities I'm afraid. Cheers Jeremy

**Comment by trizmark** [comment]
The component has been updated to include libbi support, but looks like no release has been generated.
@CJNE Any change of creating a new release for 0.0.24?

**Reply by plord12** [reply]
You can select "main" when installing ( thats what I did anyways ).

<img width="668" alt="Screenshot 2023-10-10 at 15 03 21" src="https://github.com/CJNE/ha-myenergi/assets/63543143/a07a07d5-15cb-4e27-b399-409f1f63808f">

**Comment by trizmark** [comment]
Huh, I never noticed that! Good to know, thanks!

**Comment by trizmark** [comment]
0.0.24 just been release, so HACS should show that there's an update.

**Comment by CJNE** [comment]
Sorry for the delay, swamped at work currently!
I've released 0.0.24 now with Libbi suport :)

**Comment by JeremyRoebuck85** [comment]
0.0.24 working fine here. Many thanks for including Libbi
